SwrveSDKDemo/src/main/java/com/swrve/sdk/demo/DemoApplication.java

```diff
@@ -13,7 +16,24 @@
     public void onCreate() {
         super.onCreate();
         try {
-            SwrveSDK.createInstance(this, YOUR_APP_ID, YOUR_API_KEY);
+            SwrveConfig config = new SwrveConfig();
+            // Comment the following line if you do not want to do GCM push notifications
+            config.setSenderId(YOUR_SENDER_ID);
+
+            SwrveSDK.createInstance(this, YOUR_APP_ID, YOUR_API_KEY, config);
+
+            // Comment the following lines if you do not want to do GCM push notifications
+            // React to the push notification when the user clicks on it
+            SwrveSDK.setPushNotificationListener(new ISwrvePushNotificationListener() {
```

Comment:
They usually want to react to the push payload to move to a different part of the app etc